src/impls.rs

```diff
@@ -347,6 +352,26 @@ where
     }
 }
 
+impl<Idx> TypeInfo for Range<Idx>
+where
+    Idx: TypeInfo + 'static + PartialOrd + core::fmt::Debug,
```

Comment:
We should be consistent and use `fmt::Debug` from the prelude, also for other occurrences of `core::fmt::Debug`